commands/remove.rb

```diff
@@ -54,7 +54,7 @@ def self.remove(pkg, verbose)
     device_json['installed_packages'].delete_if { |entry| entry['name'] == pkg.name }
 
     # Update device.json with our changes.
-    save_json(device_json)
+    File.write File.join(CREW_CONFIG_PATH, 'device.json'), JSON.pretty_generate(device_json.to_json)
```

Comment:
Why change this?

Reply:
The previous code was failing in certain cases-- see attatched issue.

Reply:
So the `save_json` function isn't working and that's the root cause?

Reply:
The `save_json` function isn't working in this specific case-- not sure why it was working when we weren't running things from `lib/fixup.rb`. If I had to guess, its probably something to do with the fact that `save_json` isn't actually available in `commands/remove.rb` and is only there due to the leakage of us calling it from `bin/crew`, and that doesn't happen when we call it from `lib/fixup.rb` inside `bin/crew` despite not `require`'ing the function.

This is why I'm trying to split everything out into files, and then use files in `lib` for shared function implementations. Otherwise debugging these sort of issues gets pretty complicated.

Reply:
I don't think there's a problem with `save_json`-- the problem is that we're using `save_json` at all. 

As I said:
> If I had to guess, its probably something to do with the fact that save_json isn't actually available in commands/remove.rb and is only there due to the leakage of us calling it from bin/crew, and that doesn't happen when we call it from lib/fixup.rb inside bin/crew despite not require'ing the function.